### ISSUE TITLE
Community Diligence Review of GreaterHeat Allocator

### ISSUE THREAD

**Issue by filecoin-watchdog** [filecoin-project/Allocator-Governance#14]
Review of GreaterHeat Allocations from @alangreaterheat
Allocator Application: https://github.com/filecoin-project/notary-governance/issues/1041

First example:
DataCap was given to:
https://github.com/AlanGreaterheat/Greaterheat-Allocator/issues/7

Public Open Dataset - key compliance requirement: Retrievability

1st point)
Allocation schedule per allocator:
First Round: 512 TiB
Second Round: 1 PiB
Third Round: 2 PiB
Fourth Round: 2 PiB
Overall Cap for Each Client: (2 PiB)

Actual allocation: 500Tib, 1PiB - this follows the guidelines

2nd point)
No sign of KYC or KYB of client or dataset as mentioned in allocator application. 

3rd point)
Client said these were the SPs, 
f03044609 Kenny Korea
f02953066 bluesky USA
f02837684 Xunlei HK

SPs to be prepared soon：
f02956073 Alice Singapore
f03028412 ZC Guangdong

Actual data storage report:
https://check.allocator.tech/report/AlanGreaterheat/Greaterheat-Allocator/issues/7/1715347592512.md

Provider | Location | Total Deals Sealed | Percentage | Unique Data | Duplicate Deals
f03028412 | Guangzhou, Guangdong, CNHuawei Cloud Service data center | 44.78 TiB | 8.68% | 44.78 TiB | 0.00%
f02953066 | Los Angeles, California, USIPTELECOM ASIA | 148.47 TiB | 28.77% | 148.47 TiB | 0.00%
f02837684 | Hong Kong, Hong Kong, HKIPTELECOM Global | 149.94 TiB | 29.06% | 149.94 TiB | 0.00%
f02956073 | Singapore, Singapore, SGIPTELECOM Global | 49.97 TiB | 9.68% | 49.97 TiB | 0.00%
f03044609 | Paripark, Seoul, KRThe Constant Company, LLC | 122.81 TiB | 23.80% | 122.81 TiB | 0.00%

5 of 5 SPs listed match those taking deals per report. Additional diligence needed to confirm entity and actual storage locations

4th point)
Second allocation awarded to this client.
However, per [Spark dashboard](https://bajtos.grafana.net/public-dashboards/9af156c3266d4d8384385623d0d2a76c?orgId=1&from=1709477321031&to=1713841165807), all SPs are either not available or have 0% retrievability.

The Allocator showed no sign of diligence after 1st allocation and gave the 2nd allocation of 1PiB to the client.

**Comment by Kevin-FF-USA** [comment]
Hi @AlanGreaterheat

On the next Fil+ Allocator meeting we will be going over each refill application. Wanted to ensure you were tracking the review discussion taking place in https://github.com/filecoin-project/Allocator-Governance/issues/14.

If your schedule allows, recommend coming to the May 28th meeting to answer/discuss the issues raised in the recent distributions. This will allow you to faster address - or, the issue in Allocator Governance for ongoing written discussion.

Warmly,
-Kevin
https://calendar.google.com/calendar/embed?src=c_k1gkfoom17g0j8c6bam6uf43j0%40group.calendar.google.com&ctz=America%2FLos_Angeles

![image](https://github.com/filecoin-project/Allocator-Governance/assets/94004586/cc292e57-d3fa-4739-848d-3132097385d5)

**Comment by galen-mcandrew** [comment]
Based on an additional compliance review, it appears this allocator is attempting to work with public open dataset clients. 

**However, the data associated with this pathway is not currently able to be retrieved at scale, and testing for retrieval is currently noncompliant.**

As a reminder, the allocator team is responsible for verifying, supporting, and intervening with their clients. If a client is NOT providing accurate deal-making info (such as incomplete or inaccurate SP details) or making deals with noncompliant unretrievable SPs, then the allocator needs to intervene and require client updates before more DataCap should be awarded.

Before we will submit a request for more DataCap to this allocator, please verify that you will **instruct, support, and require** your clients to work with retrievable storage providers.

**@AlanGreaterheat can you verify that you will enforce retrievability requirements, such as through Spark?** Please reply here with acknowledgement and any additional details for our review.

**Comment by AlanGreaterheat** [comment]
Of course, we will follow the official requirements for Spark retrieval rate compliance requirements (after the launch of the Spark tool), and at the same time, we also require clients to contact partner SPs to communicate to improve the Spark retrieval success rate.
<img width="774" alt="image" src="https://github.com/filecoin-project/Allocator-Governance/assets/110453718/b72f3bd6-4e2e-4bc5-bfce-49b05ed05735">


Happily, the Spark retrieval rate is steadily improving in recent stages.

last week
<img width="782" alt="image" src="https://github.com/filecoin-project/Allocator-Governance/assets/110453718/7546382a-b0a6-420f-b4aa-02aafce4f9cf">

now
<img width="730" alt="image" src="https://github.com/filecoin-project/Allocator-Governance/assets/110453718/467ccb5f-3b52-466c-af08-00b3325bd007">

**Comment by AlanGreaterheat** [comment]
According to today's latest governance meeting, we made a live speech, and based on Kevin-FF's suggestion, before the governance team further refreshes Datacap, as Greaterheat deeply rooted in the Filecoin ecosystem, we would like to clarify again that we hope to obtain a Datacap quota of 10PiB.

Regarding the second point of the report: No sign of KYC or KYB of client or dataset as mentioned in allocator application.

In fact, we have conducted multiple rounds of investigations, and we also pay attention to the past reputation of GitHub accounts.
<img width="929" alt="image" src="https://github.com/filecoin-project/Allocator-Governance/assets/110453718/7bcff531-afdd-4249-a263-d0c61e53f93f">
<img width="934" alt="image" src="https://github.com/filecoin-project/Allocator-Governance/assets/110453718/f74f12bb-00cc-4814-89cf-0fc16faf28cc">


Regarding the third point of the report: As for the geographical location, the accuracy is 100%, and we even conducted random email location confirmations.
<img width="933" alt="image" src="https://github.com/filecoin-project/Allocator-Governance/assets/110453718/b774c2f4-3a41-4f8c-a078-bd78d578300e">


Regarding the fourth point of the report: Regarding the Spark retrieval rate, before officially launching the Spark retrieval rate, we have already allocated according to the original checker: manualTrigger display report, and the report shows no issues.
<img width="944" alt="image" src="https://github.com/filecoin-project/Allocator-Governance/assets/110453718/e550bcff-7f50-4439-9882-5b76ad27c3e3">


After participating in the governance meetings of the past two weeks, we requested clients to contact SPs and research the Spark retrieval principle. It can be seen that the Spark retrieval rate has improved, some even reaching 68.71%, which is about to meet the governance team's requirement of 85%.
<img width="1105" alt="image" src="https://github.com/filecoin-project/Allocator-Governance/assets/110453718/ebf5afe1-9bc3-4f8a-bdcf-5d77be362019">


At the same time, during the feedback process with the Spark team, the SP technical personnel acknowledged that Spark still needs to be upgraded to work in coordination with the allocator path.
<img width="392" alt="image" src="https://github.com/filecoin-project/Allocator-Governance/assets/110453718/0a5139a4-c275-43cf-a05a-fa0364f97801">


In conclusion, the improvement of the fil-plus ecosystem requires efforts from all parties. In the subsequent 10PiB DC allocation process, Greaterheat will focus on data retrieval, closely monitor SP disclosure, and make the fil-plus ecosystem more prosperous.

**Comment by AlanGreaterheat** [comment]
@galen-mcandrew @Kevin-FF-USA 

I would like to follow up on this. Appreciate to let me know anything else I need to provide to expedite this request for DC refill.It is encouraging to note that the success rate of SP spark searches is increasing.

<img width="1134" alt="image" src="https://github.com/filecoin-project/Allocator-Governance/assets/110453718/c6e8212c-4844-4982-b5ae-b7704d80554a">

**Comment by AlanGreaterheat** [comment]
@galen-mcandrew @Kevin-FF-USA  Another update based on the latest governance meeting, Spark retrieval rates for nodes are improving, looking forward to good news, thanks!

<img width="1143" alt="image" src="https://github.com/filecoin-project/Allocator-Governance/assets/110453718/96240e22-8444-48bc-9aa0-efe6303b52ed">
